napari/_qt/layer_controls/qt_points_controls.py

```diff
@@ -90,6 +90,11 @@ def __init__(self, layer):
         self.layer.text.events.visible.connect(self._on_text_visibility_change)
 
         sld = QSlider(Qt.Horizontal)
+        sld.setToolTip(
+            trans._(
+                "Change the size of the currently selected points and the future ones."
```

Comment:
```suggestion
                "Change the size of currently selected points and any added afterwards."
```

For once, the thesaurus had a good suggestion I think. :smiley: 

I prefer not having "the" in front of currently.